stable/datadog/templates/agenthpa-rolebinding.yaml

```diff
@@ -4,7 +4,7 @@ kind: RoleBinding
 metadata:
   labels:
     app: "{{ template "datadog.fullname" . }}"
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    chart: {{ include "datadog.chart" . }}
```

Comment:
`{{ template "datadog.clusterAgent.fullname" . }` has to be changed here https://github.com/helm/charts/pull/11903/commits/e8e3405a850ea2294f73d8557c4329409fed4280#diff-b740101d9db451b7817c76af7b448639R10